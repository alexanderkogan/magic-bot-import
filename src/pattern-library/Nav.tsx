import * as React from 'react'
import { Props } from '../react-typings'

export default (props: Props) => (
    <nav>
        <ul className="obc_tabs">{props.children}</ul>
    </nav>
)
