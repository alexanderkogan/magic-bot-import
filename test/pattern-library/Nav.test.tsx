import * as React from 'react'

import Nav from '../../src/pattern-library/Nav'
import { mount } from '../test-utils'

describe('Nav', () => {
    it('renders', () => {
        const element = mount(<Nav>children</Nav>)
        expect(element).toMatchSnapshot()
    })
})
