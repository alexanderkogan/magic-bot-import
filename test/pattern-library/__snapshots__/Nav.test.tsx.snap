// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders 1`] = `
<Component>
  <nav>
    <ul
      className="obc_tabs"
    >
      children
    </ul>
  </nav>
</Component>
`;
